README.md

```diff
@@ -71,7 +71,7 @@ the COOL environment.
 
 | Name | Version |
 |------|---------|
-| terraform | ~> 1.0 |
+| terraform | ~> 1.5 |
```

Comment:
Good catch! 👍

Reply:
Don't praise me, praise `terraform-docs`!

Reply:
Can we resolve this conversation?

Reply:
Hi Team, one question, Do we see any reason to restrict what ports are being used ? Or does TCP only have a restricted usage of any ports from 5000 to 5999 ? For example, teamserver_sg.tf line 40 and windows_sg.tf line 39 appear to be open to use any port in the range 5000 to 5999.

Reply:
@dv4harr10 - TCP ports span the range 1-65535.  The assessors specifically asked for those thousand ports to be opened bidirectionally between the Windows and Teamserver instances.  Please see their request in #235.

Reply:
Hi @jsf9k , Yes I understand the range is by the client request, but my question is mainly concerning limiting unwanted exposure on the entire internet. If it's required by the client I can understand that.

Reply:
@dv4harr10 this is allowing communication between two security groups. Access from the internet would be a separate consideration.

Reply:
As @mcdonnnj stated, this change does not allow access from (or to) the internet.  It only allows Windows and Teamserver instances to communicate with each other via the specified ports.

Reply:
@jsf9k  Great Thanks for the clarification.

Reply:
@dav3r and @dv4harr10 - Please confirm that you both are OK with closing this discussion.

Reply:
Yes confirmed